src/Installers/Windows/AspNetCoreModule-Setup/ANCMIISExpressV2/ancm_iis_expressv2.wxs

```diff
@@ -56,7 +71,7 @@
                  Manufacturer="!(loc.Manufacturer)"
                  Compressed="yes"
                  InstallPrivileges="elevated"
-                 InstallerVersion="400"
```

Comment:
review: should we only require 500 for the arm64 installer?

Reply:
yes